pkg/util/cfg/files.go

```diff
@@ -92,13 +93,18 @@ func YAMLFlag(args []string, name string) Source {
 		if f == nil || f.Value.String() == "" {
 			return nil
 		}
-		expandEnv := false
-		expandEnvFlag := freshFlags.Lookup("config.expand-env")
-		if expandEnvFlag != nil {
-			expandEnv, _ = strconv.ParseBool(expandEnvFlag.Value.String()) // Can ignore error as false returned
-		}
-
-		return YAML(f.Value.String(), expandEnv)(dst)
 
+		for _, val := range strings.Split(f.Value.String(), ",") {
+			val := strings.TrimSpace(val)
+			expandEnv := false
+			expandEnvFlag := freshFlags.Lookup("config.expand-env")
+			if expandEnvFlag != nil {
+				expandEnv, _ = strconv.ParseBool(expandEnvFlag.Value.String()) // Can ignore error as false returned
+			}
+			if _, err := os.Stat(val); err == nil {
+				return YAML(val, expandEnv)(dst)
+			}
+		}
+		return fmt.Errorf("%s does not exist", f.Value.String())
```

Comment:
```suggestion
		return fmt.Errorf("%s config does not exist, set %s for custom config path", f.Value.String(), name)
```

Reply:
Added this suggestion because, currently it just prints
`config.yaml does not exist` if expected config file not found. Better to give some context where this is coming from and how to change it.